benchmark/Microsoft.IdentityModel.Benchmarks/ReadJWETokenTests.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using BenchmarkDotNet.Attributes;
+using Microsoft.IdentityModel.JsonWebTokens;
+using Microsoft.IdentityModel.Tokens;
+
+namespace Microsoft.IdentityModel.Benchmarks
+{
+    // dotnet run -c release -f net8.0 --filter Microsoft.IdentityModel.Benchmarks.ReadTokenTests*
+
+    public class ReadJWETokenTests
```

Comment:
Why you deleted `[Config(typeof(BenchmarkConfig))]` here and in `ReadJWSTokenTests`?

Reply:
All the config is defined in one place, instead of duplicated in each class - https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/dev/benchmark/Microsoft.IdentityModel.Benchmarks/BenchmarkConfig.cs